example/pic2bin.py

```diff
@@ -0,0 +1,81 @@
+import sys
+import binascii
+import zlib
+import cv2
+import numpy as np
+
+ENTER_KEY_WIN = 13
+ENTER_KEY_LINUX = 10
+ESC_KEY = 27
+
+
+def nothing(x):
+    pass
+
+
+if len(sys.argv) <= 1:
+    print("usage: pic2bin.py <imagefile>")
+    sys.exit()
+
+windowname = sys.argv[1] + '    Enter : save    Esc : quit'
+
+img = cv2.imread(sys.argv[1], cv2.IMREAD_COLOR)
+if img is None:
+    print("Image open error")
+    sys.exit()
+gray = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
+
+height, width = gray.shape
+if height != 64:
+    print("Image height is invalid")
+    sys.exit()
+
+cv2.namedWindow(windowname, cv2.WINDOW_AUTOSIZE | cv2.WINDOW_GUI_NORMAL)
+
+lut = np.arange(256, dtype='float32')
+lut = pow((lut / 255), 2.2)
+
+binalized = np.zeros((height, width, 1), np.uint8)
+
+cumulative_err = 0
+calc_done = 0
```

Comment:
(IMO) While it may be controversial, I prefer bools for flags.

 ```suggestion
calc_done = False
```

```diff
- calc_done = 1
+ calc_done = True
```